process_subways.py

```diff
@@ -496,6 +501,14 @@ def main() -> None:
         ", ".join(sorted(bad_city_names)),
     )
 
+    if options.dump_city_list:
+        lines = sorted(
+            f"{city.name}, {city.country}"
+            f"{' [bad]' if city.name in bad_city_names else ''}\n"
```

Comment:
`[bad]` occurs twice. Maybe make it a constant?

Reply:
\+